lib/gems/pending/appliance_console/cli.rb

```diff
@@ -207,6 +218,27 @@ def set_external_db
       config.post_activation
     end
 
+    DATE_REGEXP   = /^(2[0-9]{3})-(0?[1-9]|1[0-2])-(0?[1-9]|[12][0-9]|3[01])/
+    TIME_REGEXP   = /^(0?[0-9]|1[0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])/
```

Comment:
You should be able to reference these as `ApplianceConsole::DateTimeConfiguration::DATE_REGEXP`, and `ApplianceConsole::DateTimeConfiguration::TIME_REGEXP`. That way if we ever need to change them, we only have to do it in one place.